docs/weave-tutorial/08-app.md

```diff
@@ -44,9 +46,9 @@ Previous sections are about building your own module, now we will proceed wrappi
 
 - **main.go**: main entry point for `cmd/blog` app. Contains command line tooling.
 - **Makefile**`: contains build scripting and tooling commands
-- **app**: contains blockchain application components such as `tx`, `crontask`, and `codec.proto` of the main application.
+- **app**: contains blockchain application components, such as `tx`, `crontask`, and `codec.proto`, of the main application.
```

Comment:
`- **app**: contains blockchain application components, such as `tx`, `crontask`, and `codec.proto`, of the main application
`